README.md

```diff
@@ -1,2 +1,121 @@
-# aws-nextflow-workflows
-AWS CloudFormation templates for deploying Nextflow infrastructure
+# Nextflow Workflows Infrastructure
+
+The AWS infrastructure for hosting a private instance of [Nextflow Tower](https://tower.nf/) and executing [Nextflow workflows](https://nextflow.io/) is defined in this repository and deployed using [CloudFormation](https://aws.amazon.com/cloudformation/) via [Sceptre](https://sceptre.cloudreach.com/).
+
+## Onboarding
+
+To complete these onboarding instructions, you will need a project name (_e.g._ `imcore`, `amp-ad`, `commonmind`), and a stack name, _i.e._ the project name with the suffix `-project` (_e.g._ `imcore-project`, `amp-ad-project`, `commonmind-project`).
+
+**N.B.** Anytime that `<stack_name>` apears with the angle brackets, replace this placeholder with the actual stack name without the angle brackets.
+
+1. Install [AWS CLI v2](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) and [Docker Desktop](https://www.docker.com/products/docker-desktop).
+
+2. Open a pull request on this repository in which you duplicate `config/prod/example-project.yaml` as `config/prod/<stack_name>.yaml` and follow the numbered steps listed in the file. Note that some steps are required whereas others are optional.
```

Comment:
I think we may hit the same issue as we had for the provisioner a couple of years ago.  Unfortunately not everyone is proficient with git, so we don't necessarily want that to stop people from spinning up tower project.

Reply:
We'll keep an eye on how often Git presents a barrier. My expectation is that the average Tower user will generally be more familiar with Git than the average Service Catalog user. I also don't expect new project to be deployed very often, so admins like myself can readily help out. I added a note clarifying what to do if they don't know how to create a PR.